core/controller/src/main/scala/whisk/core/controller/Controller.scala

```diff
@@ -171,8 +173,8 @@ object Controller {
 
     def main(args: Array[String]): Unit = {
         implicit val actorSystem = ActorSystem("controller-actor-system")
-        implicit val logger = new AkkaLogging(akka.event.Logging.getLogger(actorSystem, this))
-
+        implicit val logger = new ZipkinLogging(new AkkaLogging(akka.event.Logging.getLogger(actorSystem, this)))
```

Comment:
I prefer a model where the additional logging and tracing driver is done through a configuration file (or deployment parameters) and loaded _reflectively_; This means the drivers can be developed externally and loaded as desired.